shared_model/validators/transactions_collection/transactions_collection_validator.hpp

```diff
@@ -27,14 +28,27 @@ namespace shared_model {
               TransactionValidator())
           : transaction_validator_(transactions_validator) {}
 
+      // TODO: IR-1505, igor-egorov, 2018-07-05 Remove method below when
+      // proposal and block will return collection of shared transactions
       /**
        * Validates collection of transactions
        * @param transactions collection of transactions
        * @return Answer containing errors if any
        */
       virtual Answer validate(
```

Comment:
Maybe, method's implementation should be either moved back to .cpp file, or it should not be declared as ```virtual```?